Configurations/descrip.mms.tmpl

```diff
@@ -177,35 +177,44 @@ DEFINES={- our $defines = join(",",
-CPPFLAGS={- our $cppflags = join('', @{$config{cppflags}}) -}
-CPPFLAGS_Q={- (my $x = $cppflags) =~ s|"|""|g; $defines =~ s|"|""|g;
-              $x .= "/DEFINE=($defines)";
+CPPFLAGS='qual_includes'{- our $cppflags = join('', @{$config{cppflags}}) -}
+CPPFLAGS_Q={- (my $x = $cppflags) =~ s|"|""|g;
+              (my $d = $defines) =~ s|"|""|g;
               $x .= "/INCLUDE=($includes)" if $includes;
+              $x .= "/DEFINE=($d)";
               $x; -}
 CFLAGS={- join('', @{$config{cflags}}) -}
 LDFLAGS= {- join('', @{$config{lflags}}) -}
 EX_LIBS= {- join('', map { ','.$_ } @{$config{ex_libs}}) -}
-LIB_DEFINES={- join("",
-                    (map { ",$_" }
-                     @{$target{shared_defines}},
-                     'OPENSSLDIR="""$(OPENSSLDIR_C)"""',
-                     'ENGINESDIR="""$(ENGINESDIR_C)"""')) -}
-LIB_CFLAGS={- $target{lib_cflags} // "" -}
-DSO_CFLAGS={- $target{dso_cflags} // "" -}
-BIN_CFLAGS={- $target{bin_cflags} // "" -}
-NO_INST_LIB_CFLAGS={- $target{no_inst_lib_cflags} // '$(LIB_CFLAGS)' -}
-NO_INST_DSO_CFLAGS={- $target{no_inst_dso_cflags} // '$(DSO_CFLAGS)' -}
-NO_INST_BIN_CFLAGS={- $target{no_inst_bin_cflags} // '$(BIN_CFLAGS)' -}
+
+LIB_DEFINES=$(DEFINES){- join("", (map { ",$_" }
+                                   @{$target{shared_defines}},
+                                   'OPENSSLDIR="""$(OPENSSLDIR_C)"""',
+                                   'ENGINESDIR="""$(ENGINESDIR_C)"""')) -}
+LIB_CPPFLAGS=$(CPPFLAGS)/DEFINE=($(LIB_DEFINES))
+LIB_CFLAGS=$(CFLAGS){- $target{lib_cflags} // "" -}
+DSO_DEFINES=$(DEFINES){- join("", (map { ",$_" } @{$target{dso_defines}})) -}
+DSO_CPPFLAGS=$(CPPFLAGS)/DEFINE=($(DSO_DEFINES))
+DSO_CFLAGS=$(CFLAGS){- $target{dso_cflags} // "" -}
+BIN_DEFINES=$(DEFINES){- join("", (map { ",$_" } @{$target{bin_defines}})) -}
+BIN_CPPFLAGS=$(CPPFLAGS)/DEFINE=($(BIN_DEFINES))
+BIN_CFLAGS=$(CFLAGS){- $target{bin_cflags} // "" -}
+NO_INST_LIB_CFLAGS=$(CFLAGS){- $target{no_inst_lib_cflags}
+                               // $target{lib_cflags}
+                               // "" -}
+NO_INST_DSO_CFLAGS=$(CFLAGS){- $target{no_inst_dso_cflags}
+                               // $target{dso_cflags}
+                               // "" -}
+NO_INST_BIN_CFLAGS=$(CFLAGS){- $target{no_inst_bin_cflags}
+                               // $target{bin_cflags}
+                               // "" -}
 
 PERL={- $config{perl} -}
 
-# We let the C compiler driver to take care of .s files. This is done in
-# order to be excused from maintaining a separate set of architecture
-# dependent assembler flags. E.g. if you throw -mcpu=ultrasparc at SPARC
-# gcc, then the driver will automatically translate it to -xarch=v8plus
-# and pass it down to assembler.
 AS={- $config{as} -}
-ASFLAGS={- join('', @{$config{asflags}}) -}
+ASFLAGS={- join(' ', @{$config{asflags}}) -}
+ASOUTFLAG={- $target{asoutflag} -}$(OSSL_EMPTY)
+PERLASM_SCHEME= {- $target{perlasm_scheme} -}
```

Comment:
Redundant space after =?

Reply:
BTW, if you choose to do something about this, it makes more sense to wait till appveyour is done. In such case I'd approve it without having to wait even longer.